packages/new-auth-ui/src/components/landing/index.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
```

Comment:
I can't comment on them as they are binaries but you can lose most of the above assets - you obviously cloned the dev portal but not needed here. FYI you can just use the react-app-scaffolder for a new package with yarn start